sdk/dotnet/src/Microsoft.AspNetCore.Datasync.Abstractions/Microsoft.AspNetCore.Datasync.Abstractions.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	  <Authors>Microsoft Corporation</Authors>
+	  <Company>Microsoft Corporation</Company>
+	  <Copyright>Copyright (c) 2021 Microsoft Corporation.</Copyright>
+	  <Description>An ASP.NET Core based framework for synchronizing structured data between a database and a client application interfaces and enum definitions.</Description>
+	  <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+	  <IconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</IconUrl>
+	  <LangVersion>9.0</LangVersion>
+	  <NeutralLanguage>en</NeutralLanguage>
+	  <PackageId>Microsoft.AspNetCore.Datasync.Abstractions</PackageId>
+	  <PackageLicenseExpression>MIT</PackageLicenseExpression>
+	  <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+	  <PackageTags>microsoft aspnetcore cloud datasync webapi</PackageTags>
+	  <ProductName>Microsoft.AspNetCore.Datasync.Abstractions</ProductName>
+	  <ProjectUrl>https://github.com/Azure/azure-mobile-apps</ProjectUrl>
+	  <RepositoryUrl>https://github.com/Azure/azure-mobile-apps</RepositoryUrl>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Needs to be net6.0 (to match the PR for #255)

Reply:
you haven't requested that, but I have updated even language version